functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -33,6 +39,16 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         return func.HttpResponse(status_code=500)
     logging.debug("Connected to DB successfuly!")
 
+    #Redis Sever
+    try:
+        
+        rDB = redis.Redis(host='nsc-redis-dev-usw2-thursday.redis.cache.windows.net', port='6380', db=0, password='${{ secrets.ENV_REDIS_KEY }}', ssl=True) 
+        rDB.ping()

```

Comment:
Hm, I am guessing this lets user know somehow that they were/were not able to get onto the Redis DB?

Reply:
Ill add this in the next sprint...